static/js/ajax-notifications.js

```diff
@@ -39,7 +42,7 @@ function wsCheckNotifications() {
 
         // Add handling for different kinds of events. For ex: {"event": "notification", "data": {"title": ..}}
         if (msg.event === 'notification') {
-            displayPNotify(msg.data.type, msg.data.title, msg.data.body);
+            displayPNotify(msg.data.type, msg.data.title, msg.data.body, msg.data.hash);
         } else {
             displayPNotify('info', '', msg);
```

Comment:
here is the issue:

![image](https://user-images.githubusercontent.com/2620870/28018391-91d8ad5a-6553-11e7-8053-53619c4ce590.png)